services/dashboard/views/apps/apps.js

```diff
@@ -2,64 +2,78 @@
  * Licensed under the Apache License, Version 2.0
```

Comment:
This is also a candidate for scalajs otherwise the shortcuts provided below like 
```
id: {href: app.pageUrl, text: app.appId},
```
will not be type checked